Sources/Testing/ExitTests/ExitTest.swift

```diff
@@ -52,16 +52,25 @@ public typealias ExitTest = __ExitTest
 @available(*, unavailable, message: "Exit tests are not available on this platform.")
 #endif
 public struct __ExitTest: Sendable, ~Copyable {
-  /// The expected exit condition of the exit test.
-  @_spi(ForToolsIntegrationOnly)
-  public var expectedExitCondition: ExitCondition
+  /// A type whose instances uniquely identify instances of `__ExitTest`.
+  public struct ID: Sendable, Equatable, Codable {
+    /// An underlying UUID (stored as two `UInt64` values to avoid relying on
+    /// `UUID` from Foundation or any platform-specific interfaces.)
+    private var _lo: UInt64
+    private var _hi: UInt64
```

Comment:
Any reason not to store them as a tuple, matching the initializer parameter?

Reply:
Tuples don't conform to `Codable`, so we'd have to manually implement codable conformance, and I am way too lazy for that.